### PR TITLE
3621 django freshly

### PR DESCRIPTION
**Pull request by sblack-usu**
### Pull Request Checklist: 
- [x] Positive Test Case Written by Dev

- [x] Automated Testing

- [x] Sufficient User and Developer Documentation

- [x] Passing Jenkins Build

- [x] Peer Code review and approval

### Positive Test Case
1. Open the hydroshare home page with the networks panel open.  Refresh the page and take note the js and css files are retrieved through cache.  Update the FRESHLY_ASSETS_VERSION, restart hydroshare and refresh the home page.  Note the version has been updated in the request and the requests were not retrieved from cache.

### CODE REVIEW

**Comment by sblack-usu** [comment]
hs_docker_base PR https://github.com/hydroshare/hs_docker_base/pull/35

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM hydroshare/hs_docker_base:release-1.9.9
+FROM hydroshare/hs_docker_base:3621-django-freshly
```
Comment (hyi) [inline Dockerfile]: Will the tag be changed to a release version number after the hs_docker_base PR is merged and the new image is built and tagged in dockerhub?

Comment (sblack-usu) [reply Dockerfile]: yes, once the new hs_docker_base is released, I'll update it.

**Comment by sblack-usu** [comment]
@cuahsimarko - What are you referring to?

**Comment by cuahsimarko** [comment]
> @cuahsimarko - What are you referring to?

I looked into it. These have certainly all evolved since I started on the HydroShare project. The FROM line used to exist a few places, if memory serves. Approving again.

**Comment by sblack-usu** [comment]
@hyi or @cuahsimarko - could you reapprove now that the freshly tests have been excluded?

**Pull request merged**